### DISCUSSION TITLE
Group DMs

### DISCUSSION THREAD

**Discussion by Rekunan** [ppy/osu#14334] [Ideas] [answered]
When u want to set up a multi with your friends, you can just send the invite into a group dm. Or if ur playing a team tournament, it can be helpful to create a group dm of just your teammates to chat and coordinate strategy; maybe even have it where a ref can send an invite to the match to a group dm of the team through IRC.

**Comment by peppy** [comment, marked as answer]
This is being tracked on osu-web (see https://github.com/ppy/osu-web/issues/6404)
